From: Luciano Bello <luciano@debian.org>
Subject: Indicates that the version cames from a Debian Package

--- a/core/controllers/misc/get_w3af_version.py
+++ b/core/controllers/misc/get_w3af_version.py
@@ -40,5 +40,6 @@ def get_w3af_version():
     return ('w3af - Web Application Attack and Audit Framework\n'
             'Version: %s\n'
             'Revision: %s%s\n'
+            'Distribution: Debian\n'
             'Author: Andres Riancho and the w3af team.') % (vnumber, commit,
                                                             cdate)
