From: Luciano Bello <luciano@debian.org>
Subject: Indicates that the version cames from a Debian Package

--- a/core/controllers/misc/get_w3af_version.py
+++ b/core/controllers/misc/get_w3af_version.py
@@ -33,4 +33,5 @@
     return ('w3af - Web Application Attack and Audit Framework\n'
             'Version: 1.2\n'
             'Revision: %s\n'
+            'Distribution: Debian\n'
             'Author: Andres Riancho and the w3af team.') % rev
