--- a/core/controllers/dependency_check/platforms/linux.py
+++ b/core/controllers/dependency_check/platforms/linux.py
@@ -28,11 +28,10 @@ SYSTEM_NAME = 'Debian'
 PKG_MANAGER_CMD = 'sudo apt-get install'
 
 SYSTEM_PACKAGES = {
-                   'PIP': ['python-pip'],
-                   'C_BUILD': ['python2.7-dev', 'python-setuptools',
-                               'build-essential', 'libsqlite3-dev'],
-                   'GIT': ['git'],
-                   'XML': ['libxml2-dev', 'libxslt1-dev']
+                   'PIP': [],
+                   'C_BUILD': [],
+                   'GIT': [],
+                   'XML': []
                   }
 PIP_CMD = 'pip'
 
@@ -80,4 +79,4 @@ def os_package_is_installed(package_name
             return None
 
 def after_hook():
-    pass
\ No newline at end of file
+    pass
