--- a/w3af/core/data/kb/info_set.py
+++ b/w3af/core/data/kb/info_set.py
@@ -80,8 +80,8 @@ class InfoSet(object):
     MAX_INFO_INSTANCES = 30
 
     JINJA2_ENV = Environment(undefined=StrictUndefined,
-                             trim_blocks=True,
-                             lstrip_blocks=True)
+                             trim_blocks=True)
+                             #lstrip_blocks=True)
     JINJA2_ENV.filters['human_number'] = human_number
     JINJA2_ENV.filters['sample_count'] = sample_count
 
